docs/multi-datasource/client_management_design.md

```diff
@@ -39,8 +39,8 @@ This design is part of the OpenSearch Dashboards multi data source project [[RFC
 
 ### 4.0 Answer some critical design questions
 
-**1.** **How to set up connection(clients) for different datasources?**
-Similar to how current OpenSearch Dashboards talks to default OS by creating opensearch node.js client using [opensearch-js](https://github.com/opensearch-project/opensearch-js) library, for datasources we also create clients for each. Critical params that differentiate data sources are `url` and `auth`
+**1.** **How to set up connection(clients) for different data sources?**
+Similar to how current OpenSearch Dashboards talks to default OS by creating opensearch node.js client using [opensearch-js](https://github.com/opensearch-project/opensearch-js) library, for data sources we also create clients for each. Critical params that differentiate data sources are `url` and `auth`
```

Comment:
If we are splitting data sources we might as well get the other things we missed like `OS` should be OpenSearch. 

> by creating opensearch node.js client using opensearch-js library

seems a little clunky. can just say by using creating a client using opensearch-js library.

> for data sources we also create clients for each *connection*

Reply:
also is there any memory leakage with this design?

Reply:
Addressed. Gonna open another thread to discuss the memory leakage question.